### DISCUSSION TITLE
Error: Cannot set property id of #<cp> which has only a getter

### DISCUSSION THREAD

**Discussion by engmondy** [AntlerVC/firetable#995] [Support / Q&A] [answered]
When I am trying to add a row to the table it inserts 2 rows with one of them marked as an overflow row, it's like a repeated row!

This only happened with one table in my project, not all the tables.

![Screenshot from 2022-12-03 16-56-44](https://user-images.githubusercontent.com/108227192/205444432-b27e5f39-12cc-406c-bc5a-f9ab413b6b75.png)

**Comment by BeeBombshell** [comment]
Hey @engmondy, thanks for bringing this to our notice. We're working on a fix for this issue, which would be out soon. Appreciate your patience!

**Comment by BeeBombshell** [comment, marked as answer]
Hey @engmondy, this issue has now been fixed from our end. In case you are still facing any issues, let us know.
